File: 8-kyu/multiple-of-index/multiple-of-index.js
function multipleOfIndex(array) {
  // good luck
  let multiArr = [];
  for (let i = 0; i < array.length; i++) {
    if (i === 0 && array[i] === 0) {
      multiArr.push(array[i]);
    } else if (i > 0 && array[i] % i === 0) {
      multiArr.push(array[i]);
    }
  }
  return multiArr;
}

File: 8-kyu/multiple-of-index/multiple-of-index.py
def multiple_of_index(arr):
    multi_arr = []
    for i, val in enumerate(arr):
        if i == 0 and val == 0:
            multi_arr.append(val);
        elif i > 0 and val % i == 0:
            multi_arr.append(val);
    return multi_arr